src/edge/edgeManager.ts

```diff
@@ -177,6 +177,85 @@ export class EdgeManager {
         }
     }
 
+    public async loadWebView(outputChannel: vscode.OutputChannel) {
+      const panel = vscode.window.createWebviewPanel("IoTEdgeSamples", "Azure IoT Edge Samples", vscode.ViewColumn.One, {
+        enableScripts: true,
+        retainContextWhenHidden: true,
+      });
+      panel.webview.html = await this.getWebViewContent("./assets/views/gallery.html");
+      TelemetryClient.sendEvent(Constants.showSampleGalleryEvent);
+      // Handle messages from the webview
+      panel.webview.onDidReceiveMessage(async (message) => {
+        switch (message.command) {
+          case "openSample":
+            if (message.name && message.url) {
+              await vscode.commands.executeCommand("azure-iot-edge.initializeSample", message.name, message.url, message.platform);

```

Comment:
why we need to call command here? I didn't found the command is registered. Do we really need the command or just a workaround to communicate with vscode?

Reply:
Yeah, it is not needed and I removed it